queries/yang/highlights.scm

```diff
@@ -0,0 +1,29 @@
+
+(comment) @comment
+
+; Module / submodule
+["module" "submodule"] @keyword
+
+; Keywords
+(statement_keyword) @keyword
+(extension_keyword) @string.escape
+
+; Arguments
+(built_in_type) @type.builtin
+(integer) @number
+(boolean) @boolean
+(date) @number
+(range) @string.escape
+(unquoted_range) @string.escape
+(yang_version) @string.escape
+(identifier) @variable
+(node_identifier) @variable
+(glob) @string
```

Comment:
Is this node representing a regex?
If it is, it might be worth adding an injection for those nodes.

Reply:
No, glob just matches a literal `*`, but there is one YANG statement called `pattern` that does take a regex. However, currently it's just parsed as a generic statement just like everything else, and its argument is parsed as a string. I'll note it down as a future improvement.

Reply:
we have a `string.regex` hl group if you wish to add it.